libcontainer/cgroups/fs2/cpu.go

```diff
@@ -32,6 +35,23 @@ func setCpu(dirPath string, r *configs.Resources) error {
 		}
 	}
 
+	var burst string
+	if r.CpuBurst != 0 {
+		burst = strconv.FormatUint(r.CpuBurst, 10)
+		if err := cgroups.WriteFile(dirPath, "cpu.max.burst", burst); err != nil {
+			// Sometimes when the burst to be set is larger
+			// than the current one, it is rejected by the kernel
+			// (EINVAL) as old_quota/new_burst exceeds the parent
+			// cgroup quota limit. If this happens and the quota is
+			// going to be set, ignore the error for now and retry
+			// after setting the quota.
+			if !errors.Is(err, unix.EINVAL) || r.CpuQuota == 0 {
```

Comment:
I don't know whether there is a case of `unix.ENOENT` in `cgroupv2` or not? I think there is a high probability that there is no this type case. But maybe I'm wrong.

Reply:
I think it's good keep the same code style between cgroupv1 and v2

Reply:
So we need add the same logic for  'unix.ENOENT' like v1?

Reply:
Yep, I think this would be a good idea